link

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env php
+<?php
+
+/*
+* This file is part of the Symfony package.
+*
+* (c) Fabien Potencier <fabien@symfony.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+/**
+ * Link dependencies to components to a local clone of the monolithic Symfony repository.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+
+if ($argc < 2) {
+    echo 'Link dependencies to components to a local clone of the monolithic Symfony repository'.PHP_EOL.PHP_EOL;
+    echo 'Usage: link.php /path/to/the/project'.PHP_EOL;
+    exit;
+}
+
+if (!is_dir("$argv[1]/vendor/symfony")) {
+    echo "The directory \"$argv[1]\" does not exist or the dependencies are not installed (run \"composer install\"?)";
+    exit(1);
+}
+
+$sfPackages = array();
+foreach (glob(__DIR__.'/src/Symfony/{Bundle,Bridge,Component}/*', GLOB_BRACE | GLOB_ONLYDIR | GLOB_NOSORT) as $dir) {
```

Comment:
`GLOB_BRACE` doesn't work on some systems (Solaris), but I think it doesn't matter for this script.